docs/tables/aws_vpc_flow_log.md

```diff
@@ -1,31 +1,54 @@
 # Table: aws_vpc_flow_log_service
 
-VPC Flow Logs is a feature that enables to capture information about the IP traffic going to and from network interfaces in the VPC
+VPC Flow Logs is a feature that enables to capture information about the IP traffic going to and from network interfaces in the VPC.
 
 ## Examples
 
-### Flowlog delivery logs configuration details
+### List flow logs with their corresponding VPC Ids, subnet Ids, or network interface Ids
 
 ```sql
 select
-  log_group_name,
-  log_destination_type,
-  log_format,
-  flow_log_id,
-  deliver_logs_error_message,
-  deliver_logs_permission_arn,
-  deliver_logs_status
+	flow_log_id,
+	resource_id
 from
-  aws_vpc_flow_log;
+	aws_vpc_flow_log;
 ```
 
 
-### List Flowlogs with their corresponding resource IDs
+### List of flow logs whose logs delivery has failed
 
 ```sql
 select
-  flow_log_id,
-  resource_id
+	flow_log_id,
+  resource_id,
+	deliver_logs_error_message,
+	deliver_logs_status
 from
-  aws_vpc_flow_log;
+	aws_vpc_flow_log
+where
+	deliver_logs_status = 'FAILED';
+```
+
+
+### Log group or destination bucket information to which the flow log is published
+
+```sql
+select
+	flow_log_id,
+	log_destination_type,
+	log_group_name,
+	split_part(log_destination, ':', 6) as bucket_name
```

Comment:
Can we add a separate column to store the split value of `log_destination`, instead of doing it in docs?